ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/MiniProtocol/ChainSync/Client/Jumping.hs

```diff
@@ -148,33 +155,42 @@
 -- If dynamo or objector claim to have no more headers, they are disengaged
 -- (j|k).
 --
+-- The BlockFetch logic can ask to change the dynamo if it is not serving blocks
+-- fast enough. If there are other non-disengaged peers the dynamo is demoted to
+-- a jumper (l) and a new dynamo is elected.
```

Comment:
Isn't the objector also demoted in that case? Like everybody goes back to Happy jumper apart from one peer that is the new dynamo?

Reply:
That's true.

Reply:
Done in 72a5077edd144fb9aa7581cf10c50e3fa253679a.